src/components/LandingPage/LandingPage.js

```diff
@@ -9,13 +9,22 @@ import {offersAPI} from "../../utils/apis/OfferApi";
 import CenteredCircularProgress from "../commons/CenteredCircularProgress";
 import {jwtUtils} from "../../utils/jwt/jwtUtils";
 import {userTypes} from "../../utils/Enums";
+import Swal from "sweetalert2";
 
 const LandingOffers = () => {
     const history = useHistory()
     const [offers, setOffers] = useState(null)
 
     useEffect(() => {
-        return offersAPI.getAllOffers(true).then(data => setOffers(data || []));
+        offersAPI.getAllOffers(true)
+            .then(data => setOffers(data || []))
+            .catch(() => {
+                Swal.fire({
+                    title: "Error",
+                    text: "We weren't able to get offers!",
+                    icon: "error"
+                })
```

Comment:
Czy my przypadkiem nie mamy takiego ładnego obrazka? Bo swal wydaje mi się kiepskim rozwiązaniem na landing page'u?

Reply:
Mamy ładny obrazek, ale to inny kejs, od początku mamy Swala rzucającego popup jeśli wystąpił błąd przy ładowaniu ofert